### PR TITLE
fix: Org booking pages link fixes for Routing Forms and Event type

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

- Fixes booking links for Routing Forms and EventType(from Single view) for an organization.

- Fixes link and avatar(missed in avatars PR) on members page 
image

## Requirement/Documentation
All booking pages are supposed to be on organisation domain(including Routing Forms)

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
  - No Env Vars needed to test locally.
  - Simply create an event-type and a routing-form after logging in as an organization member.
  - Go to event-type e.g. /event-types/1 and click on "Preview" and check that URL is of org domain(e.g. org.cal.com)
image
- Similarly, to Routing Form e.g./apps/routing-forms/forms/1 and click on "Preview" and check that the URL is of org domain.
 image
- Do a repeat test with EventType and Routing Form for a non-org user and see that now the URL is app URL and not org url.

## Mandatory Tasks

  - [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: apps/web/components/eventtype/EventTypeSingleLayout.tsx [hunk 1]
```diff
@@ -233,9 +234,11 @@ function EventTypeSingleLayout({
     formMethods,
   ]);
 
-  const permalink = `${CAL_URL}/${team ? `team/${team.slug}` : eventType.users[0].username}/${
-    eventType.slug
-  }`;
+  const orgBranding = useOrgBrandingValues();
+
+  const permalink = `${orgBranding?.fullDomain ?? CAL_URL}/${
```
Comment (hariombalhara) [inline apps/web/components/eventtype/EventTypeSingleLayout.tsx:239]: Same logic was used in /event-types/index.tsx and it was missing here

File: packages/lib/getOrgAwareUrl.ts
Comment (hariombalhara) [inline packages/lib/getOrgAwareUrl.ts:1]: We don't want to get the URL relative to current domain for the organization. It should always be the organization URL. So, this utility is not needed.

File: packages/features/ee/teams/components/MemberListItem.tsx [hunk 2]
```diff
@@ -116,7 +118,7 @@ export default function MemberListItem(props: Props) {
           <div className="flex">
             <Avatar
               size="sm"
-              imageSrc={WEBAPP_URL + "/" + props.member.username + "/avatar.png"}
+              imageSrc={(orgBranding?.fullDomain ?? WEBAPP_URL) + "/" + props.member.username + "/avatar.png"}
```
Comment (hariombalhara) [inline packages/features/ee/teams/components/MemberListItem.tsx:121]: Fixes missing avatar

File: apps/web/components/eventtype/EventTypeSingleLayout.tsx [hunk 3]
```diff
@@ -233,9 +234,11 @@ function EventTypeSingleLayout({
     formMethods,
   ]);
 
-  const permalink = `${CAL_URL}/${team ? `team/${team.slug}` : eventType.users[0].username}/${
-    eventType.slug
-  }`;
+  const orgBranding = useOrgBrandingValues();
+  const isOrgEvent = orgBranding?.fullDomain;
+  const permalink = `${orgBranding?.fullDomain ?? CAL_URL}/${
+    team ? `${!isOrgEvent ? "team/" : ""}${team.slug}` : eventType.users[0].username
```
Comment (hariombalhara) [inline apps/web/components/eventtype/EventTypeSingleLayout.tsx:240]: An org event doesn't need team/

**Review by leog — APPROVED** [review]
Changes look good

**Review by emrysal — APPROVED** [review]

**Pull request merged**